elyra/pipeline/pipeline_definition.py

```diff
@@ -15,23 +15,26 @@
 #
 import json
 import os
-from typing import Any
+from typing import Any, Set
```

Comment:
Hmm not sure why this is passing lint? (and same with line 23 below). I'll fix it but might be worth looking into further?

Reply:
Per discussion, it appears that the linting rules that we use will allow duplicates per line for the `typing` module. I've already separated them in this PR, but something to keep in mind for the future.